### PR TITLE
cleanup: remove old code

### PR DESCRIPTION
**Pull request by justinmk3**
See commit messages.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/shared/codelens/codeLensUtils.ts [hunk 1]
```diff
@@ -252,16 +240,12 @@ export async function makeCSharpCodeLensProvider(): Promise<vscode.CodeLensProvi
...
 export function makeTypescriptCodeLensProvider(): vscode.CodeLensProvider {
-    const logger = getLogger()
-
     return {
         provideCodeLenses: async (
             document: vscode.TextDocument,
             token: vscode.CancellationToken
         ): Promise<vscode.CodeLens[]> => {
             const handlers = await tsCodelens.getLambdaHandlerCandidates(document)
-            logger.debug('makeTypescriptCodeLensProvider handlers: %O', handlers)
```
Comment (kiiadi) [inline src/shared/codelens/codeLensUtils.ts:263]: I'm all for removing code - but out of interest what's the motivation behind removing these log lines? Are they cluttering up a log file somewhere?

to put it another way - what's the cost to keeping them?

**Review by kiiadi — APPROVED** [review]

**Review by bryceitoc9 — APPROVED** [review]
🧹 🧹 🧹

File: src/shared/codelens/codeLensUtils.ts (hunk 1 above)
Comment (justinmk3) [reply src/shared/codelens/codeLensUtils.ts:263]: Yes, they are very noisy (and frequent) and not really useful. I added them awhile ago but don't need them anymore.

Good catch though, I agree in general.

**Comment by justinmk3** [comment]
Verified locally codelenses display and work correctly in all supported filetypes, so this is low-risk to merge.

**Pull request merged**